app/controllers/api/v1/api_controller.rb

```diff
@@ -3,6 +3,8 @@ module V1
     class ApiController < ApplicationController
       include DeviseTokenAuth::Concerns::SetUserByToken
       before_action :authenticate_user!
+      respond_to :json
```

Comment:
Let's see if we could remove this and try to make it works.. I think we are not using properly the `jbuilder` gem, so that's cause the issue.